PostFreeSurfer/PostFreeSurferPipelineLongLauncher.sh

```diff
@@ -57,198 +56,176 @@ fi
 opts_ShowValues
 
 log_Check_Env_Var FSLDIR
-verbose_red_echo "---> Starting ${log_ToolName}"
-verbose_echo " "
-verbose_echo " Using environment setting ..."
-verbose_echo "          HCPPIPEDIR: ${HCPPIPEDIR}"
-verbose_echo " "
-IFS=, read -r -a Timepoints <<< "${Timepoint_list[i]}"
+log_Msg "HCPPIPEDIR: ${HCPPIPEDIR}"
 
-fslsub_queue=NONE
+IFS=@ read -r -a Timepoints <<< "${Timepoint_list[i]}"
 
-case $parallel_mode in
-	FSLSUB) fslsub_queue=$queue ;;
-	BUILTIN) if (( max_jobs<1 )); then max_jobs=2; fi ;;
-	NONE) echo "parallel mode is NONE" ;;
-	*) log_Err_Abort "Unknown parallel mode. Please specify one of FSLSUB, BUILTIN, NONE." ;;
-esac
+fslsub_queue=NONE
+if [ "$parallel_mode" == "FSLSUB" ]; then
+  if [ -n "$queue" ]; then fslsub_queue=$queue; fi
```

Comment:
`opts_AddOptional` declares a default of "short.q".  Why this extra defaultish logic here?  If the mode isn't "FSLSUB", then it doesn't matter what the queue gets set to, because it doesn't get used.  Just pass whatever newopts gave you without messing with it, to simplify the logic.

Reply:
agree